distribution/docs/src/main/resources/content/_integrating/_endpoints/opensearch-endpoint.adoc

```diff
@@ -177,7 +177,7 @@ Examples:
 https://localhost:8993/services/catalog/query?q='*'&lon=44.792&lat=-6.171
 ----
 
-==== Additional OpenSearch Query Parameters
+=== Additional OpenSearch Query Parameters
```

Comment:
✏️ Move `Complex Contextual Query Format` under `OpenSearch Contextual Queries` and rename to `Complex OpenSearch Contextual Query Format`

Reply:
fixed